tools/run_tests/generated/OWNERS

```diff
@@ -0,0 +1,4 @@
+set noparent
+
+# Files in this directory are generated. Anyone from grpc-team can approve.
+@grpc/grpc-team
```

Comment:
I think this is the correct way to tell github that that anyone on the team can approve.
`@grpc/grpc-team` corresponds to https://github.com/orgs/grpc/teams/grpc-team.
If the PR starts spamming people once changes are being made to `run_tests/generated`, I will revert the PR.